src/Database/Concerns/BelongsToTenant.php

```diff
@@ -21,7 +21,11 @@ public function tenant(): BelongsTo
 
     public static function bootBelongsToTenant(): void
     {
-        static::addGlobalScope(new TenantScope);
+        // If 'tenancy.rls.enabled' is true or this model implements RLSModel
+        // Scope queries using Postgres RLS instead of TenantScope
```

Comment:
```suggestion
        // If 'tenancy.rls.enabled' is true or this model implements RLSModel
        // Postgres RLS is used for scoping, so we don't enable the scope used with single-database tenancy.
```
Also just to confirm — this means that if `tenancy.rls.enabled` is used, individual models don't need to implement `RLSModel`, and RLS is used as the default "strategy" for scoping models that use this trait, right?

Reply:
Yeah, that's correct